test/messaging.go

```diff
@@ -9,11 +9,22 @@ import (
 	"github.com/influxdb/influxdb/messaging"
 )
 
+var (
+	testDataURL *url.URL
+)
```

Comment:
Actually, why is `testDataURL` a global? Can you add a function `MustParseURL()` and put it wherever it's used? That way people don't have to jump around to understand a test function.